graylog2-server/src/main/java/org/graylog2/contentpacks/model/entities/KeywordRangeEntity.java

```diff
@@ -44,6 +49,12 @@ public static KeywordRangeEntity of(KeywordRange keywordRange) {
         return new AutoValue_KeywordRangeEntity.Builder();
     }
 
+    @Override
+    public final TimeRange convert(Map<String, ValueReference> parameters) throws InvalidRangeParametersException {
```

Comment:
How about catching `InvalidRangeParametersException` and throwing some `Runtimeexception` instead? This would remove the need to declare (or handle) in a bunch of places in `DashboardWidgetConverter`. 
IMO `InvalidRangeParametersException` should be an unchecked exception anyway as nothing can be done about it by the system at runtime, but that's a different story and out of scope for this PR of course.

Reply:
Same applies to the other `*RangeEntity` classes of course.

Reply:
:+1: